src/libstore/build/derivation-goal.cc

```diff
@@ -911,8 +911,15 @@ void DerivationGoal::buildDone()
                     msg += line;
                     msg += "\n";
                 }
-                msg += fmt("For full logs, run '" ANSI_BOLD "nix log %s" ANSI_NORMAL "'.",
-                    worker.store.printStorePath(drvPath));
+                msg += "For full logs, run '" ANSI_BOLD;
+                if (experimentalFeatureSettings.isEnabled(Xp::NixCommand)) {
+                    msg += "nix log";
+                } else {
+                    msg += "nix-store -l";
+                }
+                msg += " ";
+                msg += worker.store.printStorePath(drvPath);
+                msg += ANSI_NORMAL "'.";
```

Comment:
Not a big deal, so feel free to ignore it if you don't like it, but I'd rather keep the previous `fmt` format as it's easier to read:

```suggestion
                nixLogCommand = experimentalFeatureSettings.isEnabled(Xp::NixCommand)
                    ? "nix log"
                    : "nix-store -l";
                msg += fmt("For full logs, run '" ANSI_BOLD "%s %s" ANSI_NORMAL "'.",
                    nixLogCommand,
                    worker.store.printStorePath(drvPath));
```

Reply:
That's much better, thanks (updated).